src/store/db.ts

```diff
@@ -47,7 +48,8 @@ export const getEpoch = async (arIOReadSDK: AoARIORead, epochIndex: number) => {
 
   if (epochData) {
     try {
-      await db.epochs.add(epochData);
+      console.log('Saving epoch', epochIndex, epochData)
```

Comment:
intended to leave log?

Reply:
Did not, removed now, thanks for spotting